apps/web/src/app/layout.tsx

```diff
@@ -11,10 +11,10 @@ import { Metadata } from "next";
 import socialpreview from "@/assets/social-preview.png";
 
 export const metadata: Metadata = {
+  metadataBase: new URL("https://jeteo.newbies.pl"),
```

Comment:
tu prośba o wyciągnięcie tego metadatabase do zmiennej środowiskowej, jak na sztywno dasz adres proda to lokalnie nie będzie to działać